app/views/layouts/application.html.erb

```diff
@@ -2,9 +2,7 @@
 <html>
 <head>
   <title>Taxiproject</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```

Comment:
But it doesn't work after removal of gem "coffee-rails". I have not figured out with this